src/uct/api/uct.h

```diff
@@ -789,6 +791,32 @@ ucs_status_t uct_mem_alloc(size_t min_length, uct_alloc_method_t *methods,
  */
 ucs_status_t uct_mem_free(const uct_allocated_memory_t *mem);
 
+/**
+ * @ingroup RESOURCE
+ * @brief Read the configuration of the PD component.
+ *
+ * @param [in]  name          Name of the PD or the PD component.
+ * @param [in]  env_prefix    If non-NULL, search for environment variables
+ *                            starting with this UCT_<prefix>_. Otherwise, search
+ *                            for environment variables starting with just UCT_.
+ * @param [in]  filename      If non-NULL, read configuration from this file. If
+ *                            the file does not exist, it will be ignored.
+ * @param [out] config_p      Filled with a pointer to the configuration.
```

Comment:
the env_prefix and filename parameters are not used currently (and weren't used before i changed the code).
we can add more documentation when they are used later on.